telescopebe/telescopebe.pro

```diff
@@ -0,0 +1,43 @@
+TEMPLATE = lib
```

Comment:
I don't get the point of this telescope backend module. I would actually prefer merging that with telescope like what starfish does. It will probably require to use an actual main and no qmlscene.

Reply:
I use clickable https://github.com/bhdouglass/clickable to build and run the app from my Ubuntu 17.10 development system. The ubuntu sdk doesen't work under 17.10. Clickable provide this project setup as template. I also tried a version with a main.cpp but I didin't make it work. But it should work with ubuntu touch. I will ask the developer of clickable.

Reply:
Now the main.cpp version works.